qemu/tests/cfg/fio_linux.cfg

```diff
@@ -0,0 +1,22 @@
+- fio_linux:
+    type = fio_linux
+    only Linux
+    only virtio_blk virtio_scsi
```

Comment:
Please add spapr_vscsi, the test could PASS:
 (1/2) Host_RHEL.m8.u0.qcow2.spapr_vscsi.up.virtio_net.Guest.RHEL.8.1.0.ppc64le.io-github-autotest-qemu.fio_linux.aio_native: PASS (390.53 s)
 (2/2) Host_RHEL.m8.u0.qcow2.spapr_vscsi.up.virtio_net.Guest.RHEL.8.1.0.ppc64le.io-github-autotest-qemu.fio_linux.aio_threads: PASS (437.30 s)

Reply:
thanks , have updated.